### PR TITLE
dev-libs/jansson: Bump to version 2.9 (bug #598832) and to EAPI-6.

### PR DESCRIPTION
**Pull request by Polynomial-C**
Added multilib support (bug 570006).
This release also fixes the security issue reported in bug #581954.

Please review and give ACK/nACK. I'd like to `git am` this myself.

### CODE REVIEW

File: dev-libs/jansson/jansson-2.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+	if multilib_is_native_abi && use doc ; then
+		HTML_DOCS=( ${BUILD_DIR}/doc/_build/html/* )
+	fi
+}
+
+multilib_src_install_all() {
+	DOCS=( CHANGES README.rst )
+	default
+}
```
Comment (SoapGentoo) [inline dev-libs/jansson/jansson-2.9.ebuild]: 1) don't call default in `multilib_src_install_all`
2) you likely don't need `multilib_src_install_all` at all, as `einstalldocs` should catch all your `DOCS`

File: dev-libs/jansson/jansson-2.9.ebuild [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	if multilib_is_native_abi && use doc ; then
+		emake html
+	fi
+}
+
+multilib_src_install() {
+	default
+
+	if multilib_is_native_abi && use doc ; then
+		HTML_DOCS=( ${BUILD_DIR}/doc/_build/html/* )
```
Comment (SoapGentoo) [inline dev-libs/jansson/jansson-2.9.ebuild]: `HTML_DOCS=( ${BUILD_DIR}/doc/_build/html/. )` rather (avoid globbing is possible)

File: dev-libs/jansson/jansson-2.9.ebuild [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	econf \
+		$(use_enable static-libs static)
+}
+
+multilib_src_compile() {
+	default
+
+	if multilib_is_native_abi && use doc ; then
+		emake html
+	fi
```
Comment (SoapGentoo) [inline dev-libs/jansson/jansson-2.9.ebuild]: `multilib_is_native_abi && use doc && emake html`

Comment (SoapGentoo) [inline dev-libs/jansson/jansson-2.9.ebuild]: this one's not important, calling `default` in `multilib_src_install_all` is wrong though

**Comment by Polynomial-C** [comment]
Requested changes have been incorporated...

**Comment by SoapGentoo** [comment]
@gktrk Please ACK, due to inactivity by the proxy maintainer.

**Pull request closed without merging**